### PR TITLE
[FOR RE-REVIEW] fix greenrush issues

### PR DESCRIPTION
**Pull request by jrleonardo**
Description:
Fix greenrush issue on not getting transaction data for v1 and v2.

### CODE REVIEW

**Review by pacholoamit — APPROVED** [review]
Hello, looks good.

**Review by vancebaritugo — CHANGES_REQUESTED** [review]

File: src/v1/imports/carts/greenrush.ts [hunk 1]
```diff
@@ -31,16 +25,18 @@ const greenrushTracker = ({
...
               "addItem",
-              transaction.id,
-              item.id,
+              transaction.data.id.toString(),
+              item.id.toString(),
               item.name,
-              item.subcategory,
+              item.category,
               item.price,
               item.quantity,
```
[the remark this answers is not in the document]
Comment (jrleonardo) [reply src/v1/imports/carts/greenrush.ts]: applied error handling

File: src/v1/imports/carts/greenrush.ts [hunk 2]
```diff
@@ -31,16 +25,18 @@ const greenrushTracker = ({
...
               "US"
```
[the remark this answers is not in the document]
Comment (jrleonardo) [reply src/v1/imports/carts/greenrush.ts]: done changing to USD

File: src/v1/imports/carts/greenrush.ts [hunk 3]
```diff
@@ -31,16 +25,18 @@ const greenrushTracker = ({
...
             );
           }
           window.tracker("trackTrans");
+        } else {
+          return;
```
[the remark this answers is not in the document]
Comment (jrleonardo) [reply src/v1/imports/carts/greenrush.ts]: removed else statement

**Review by vancebaritugo — APPROVED** [review]

**Pull request merged**